### ISSUE TITLE
subdivide_loop_vedo produces empty mesh

### ISSUE THREAD

**Issue by haesleinhuepf** [haesleinhuepf/napari-process-points-and-surfaces#57]
... not sure why.

To reproduce:
```
import napari_process_points_and_surfaces as nppas
gastruloid = nppas.gastruloid()

subdivided_gastruloid = nppas.subdivide_loop_vedo(gastruloid, number_of_iterations=1)
subdivided_gastruloid 
```

![image](https://user-images.githubusercontent.com/12660498/219864334-6f26186a-a448-4cec-831d-5737deab80c0.png)

Workarounnd: install open3d and use this:

```
import napari_process_points_and_surfaces as nppas
gastruloid = nppas.gastruloid()

subdivided_gastruloid = nppas.subdivide_loop(gastruloid, number_of_iterations=1)
nppas.SurfaceTuple(subdivided_gastruloid)
```

![image](https://user-images.githubusercontent.com/12660498/219864409-adf42669-64f6-4290-9f21-72930a904c56.png)

**Comment by haesleinhuepf** [comment]
`nppas.subdivide_linear` and `subdivide_butterfly` appear to have the same issue.

`nppas.subdivide_adaptive` appears to work though.

**Comment by haesleinhuepf** [comment]
Hey Johannes @jo-mueller ,

would you mind taking care of this issue? Also see the related PR. Feel free to use its branch to test things.

Thanks!

**Comment by jo-mueller** [comment]
Hi @haesleinhuepf ,

I can have a look. Judging from what @marcomusy suggested [here](https://github.com/marcomusy/vedo/issues/813#issuecomment-1436747337), would it be an option to implement the vedo [isosurface](https://vedo.embl.es/docs/vedo/base.html#BaseGrid.isosurface) method in nppas as an option to marching cubes (as opposed to augmenting marching cubes with multiple failsafes)? 

Nonetheless, these failsafes should ultimately be available, too.

**Comment by haesleinhuepf** [comment]
> implement the vedo [isosurface](https://vedo.embl.es/docs/vedo/base.html#BaseGrid.isosurface) method in nppas as an option to marching cubes

Yes, let's do this step by step. If nppas has a `isosurface` function, we can play with it. If I understand it right, it is an alternative to marching cubes, no? If it does its job well, we could use it, recommend it in the documentation and then deprecate the partially broken marching cubes.
